### ISSUE TITLE
About glint dataset

### ISSUE THREAD

**Issue by Edwardmark** [deepinsight/insightface#303]
I fined tune r50 model with glint data all with triplet loss or arcface loss. The training log showed 99.8% acc in LFW, 94.50% in CFP and 98.10% in AgeDB, but when I submit the testdata feature of glint data, the result is only 20%, much lower than r50 (without fine-tuned) 48%, anyone knows why? Or is there anybody who submit results fine-tuned in glint training data?

**Comment by wenjie710** [comment]
What's the r50 model you use? Is it the released pre-trained model LResNet50E-IR@BaiduDrive?

**Comment by Edwardmark** [comment]
@wenjie710 yes

**Comment by wenjie710** [comment]
Do you use src/eval/gen_glint.py to generate the .bin file? I use that to generate testdata feature of glint data but the result I get is only 31%. I don't know why.

**Comment by Edwardmark** [comment]
@wenjie710 yes, I use gen_glint.py to get the bin file

**Comment by Edwardmark** [comment]
@wenjie710 Did you use the right lmk file and lst file?
